docs/5.0/preview/upcoming-releases.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Teleport Upcoming Releases
+description: A timeline of teleport upcoming releases.
+---
+
+# Upcoming Teleport Releases
+
+The teleport team delivers a new major release every 3 months.
```

Comment:
Kubernetes has been doing 4 releases per year for years, and now is moving to [3 releases per year](https://github.com/kubernetes/sig-release/issues/1290) since users find it difficult to keep up (we know this from first hand experience). Not saying we should do this now, just something to think about for future.

Reply:
That's probably because it's so hard to upgrade as you know :) Hopefully our #3121 will let us push releases even sooner.